docs/structured-streaming-programming-guide.md

```diff
@@ -2824,12 +2824,6 @@ df.writeStream
   .trigger(Trigger.ProcessingTime("2 seconds"))
   .start()
 
-// One-time trigger
-df.writeStream
-  .format("console")
-  .trigger(Trigger.Once())
-  .start()
-
```

Comment:
Hmm, but we still have "One-time micro-batch" in the doc. Do you want to remove it or mark it as deprecated too?

Reply:
Yeah not sure what would be a good way to do. While we can do aggressively via removing all occurrences about Trigger.Once, probably marking as deprecated without removing the content might make more sense as we still have users leveraging Trigger.Once.

Reply:
I just dealt with mentioning deprecation, and more information in Trigger.AvailableNow to produce motivation to migrate.